var/spack/repos/builtin/packages/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Watch(Package):
```

Comment:
This should be converted to use the `AutotoolsPackage` base class.

Reply:
I was trying to get that to work, but how can I override the fact that it does a `make install` in that package?

Reply:
If you just override the `install` method as you did, it should skip the `make install` default implementation and take yours :)